README.md

```diff
@@ -98,10 +98,25 @@ _Tools, software, and other resources related to blockchain governance in the IC
 ## Interoperability
 _Tools and software related to blockchain interoperability in the ICON ecosystem_
 
-- [btp](https://github.com/icon-project/btp) - Blockchain Transmission Protocol, which is the ICON Foundation's core interoperability product
+- [btp](https://github.com/icon-project/btp2) - Blockchain Transmission Protocol, which is the ICON Foundation's core interoperability product
 - [btp-litepaper](https://github.com/icon-project/btp-litepaper) - Lite explainer paper for BTP. [Also available here](https://icon.community/assets/btp-litepaper.pdf)
+- [BTP IIP Standard](https://github.com/icon-project/IIPs/blob/master/IIPS/iip-52.md) - ICON BTP Arbitrary Call Service Standard.
```

Comment:
This is the xCall IIP Standard. BTP IIP standard is here: https://github.com/icon-project/IIPs/blob/master/IIPS/iip-25.md